### DISCUSSION TITLE
Pattern lines are not showing up on my mat

### DISCUSSION THREAD

**Discussion by Janetk5** [Pattern-Projector/pattern-projector#354] [Q&A]
I have been using Pattern Projector successfully for several months. Today when I went to project the pattern, only the layers tab and the pattern parts labels show up, but not the actual lines to cut along. I tried rebooting my computer but it's still doing the same thing. I also re-downloaded the pattern, tried opening other patterns, with no success. Can anyone help me?

**Comment by courtneypattison** [comment]
@Janetk5 sorry for this issue! Are you able to share a screenshot showing what you see when loading a pattern? Here are a few suggestions:

Try a different browser: Chrome, Edge, and Firefox work best
Try a different version of PP: there's beta.patternprojector.com (it has the newest features) and old.patternprojector.com (it's the previous version of the site before the latest update a few weeks ago)

**Comment by sashasewist** [comment]
@Janetk5 which pattern was it? This problem can sometimes be fixed by using line width tool.

**Comment by Janetk5** [comment]
It is the meadow dress by peek-a-boo patterns but it worked fine a couple of days ago. And then I tried another pattern, the shirt jacket, that I used last week and the same thing happened. When I hover over the notations on the pattern pieces I can see them. I can also see the calibration lines. Sent from my iPadOn Nov 16, 2024, at 7:46 PM, sashasewist ***@***.***> wrote:﻿
@Janetk5 which pattern was it? This problem can sometimes be fixed by using line width tool.

**Comment by Janetk5** [comment]
Thank you! I will try the newer version of pattern projector tomorrow morning and let you know what happens. I’ll send a screenshot too.Sent from my iPadOn Nov 16, 2024, at 6:39 PM, Courtney Pattison ***@***.***> wrote:﻿
@Janetk5 sorry for this issue! Are you able to share a screenshot showing what you see when loading a pattern? Here are a few suggestions:
Try a different browser: Chrome, Edge, and Firefox work best
Try a different version of PP: there's beta.patternprojector.com (it has the newest features) and old.patternprojector.com (it's the previous version of the site before the latest update a few weeks ago)

**Comment by Janetk5** [comment]
I’m using chrome on a MacBook Air. I’m attaching a screenshot Sent from my iPhoneOn Nov 16, 2024, at 11:03 PM, Janet Kowal ***@***.***> wrote:﻿It is the meadow dress by peek-a-boo patterns but it worked fine a couple of days ago. And then I tried another pattern, the shirt jacket, that I used last week and the same thing happened. When I hover over the notations on the pattern pieces I can see them. I can also see the calibration lines. Sent from my iPadOn Nov 16, 2024, at 7:46 PM, sashasewist ***@***.***> wrote:﻿
@Janetk5 which pattern was it? This problem can sometimes be fixed by using line width tool.

**Comment by Janetk5** [comment]
Update: I used old.patternprojector.com and now it’s working again. Thanks for your help!Sent from my iPhoneOn Nov 16, 2024, at 11:03 PM, Janet Kowal ***@***.***> wrote:﻿It is the meadow dress by peek-a-boo patterns but it worked fine a couple of days ago. And then I tried another pattern, the shirt jacket, that I used last week and the same thing happened. When I hover over the notations on the pattern pieces I can see them. I can also see the calibration lines. Sent from my iPadOn Nov 16, 2024, at 7:46 PM, sashasewist ***@***.***> wrote:﻿
@Janetk5 which pattern was it? This problem can sometimes be fixed by using line width tool.

**Comment by courtneypattison** [comment]
@Janetk5 glad you got it working! Did you try beta.patternprojector.com ? I'm curious because old will get updated with the next release so I want to make sure it'll work in the future when I update. Also, I don't see the screenshot. Are you able to send it to courtney@patternprojector.com if you can't attach it here? Thank you!

**Comment by Janetk5** [comment]
The beta didn’t work for me. Only the old worked. Sent from my iPhoneOn Nov 17, 2024, at 12:47 PM, Courtney Pattison ***@***.***> wrote:﻿
@Janetk5 glad you got it working! Did you try beta.patternprojector.com ? I'm curious because old will get updated with the next release so I want to make sure it'll work in the future when I update. Also, I don't see the screenshot. Are you able to send it to ***@***.*** if you can't attach it here? Thank you!

**Comment by Janetk5** [comment]
I’m assuming that my MacBook is too old to update Chrome, and therefore too old for the newer version of Patternprojector. Just ordered a newer MacBook and I think that should solve my problem. It was time anyway!!
Thanks for your help and I love your software.
Janet
